capellambse/_namespaces.py

```diff
@@ -3,28 +3,63 @@
 
 from __future__ import annotations
 
+__all__ = [
+    "NAMESPACES",
+    "Plugin",
+    "UnsupportedPluginError",
+    "UnsupportedPluginVersionError",
+    "check_plugin",
+    "get_keys_and_plugins_from_namespaces_by_url",
+]
+
 import collections.abc as cabc
 import dataclasses
 import re
-import typing as t
 
 PLUGIN_PATTERN = re.compile(r"(.*?)(\d\.\d\.\d)?$")
 
 
-class UnsupportedVersionError(ValueError):
-    """Raised when plugin version is outside of ["5.0.0", "6.0.0"]."""
+class UnsupportedPluginError(ValueError):
+    """Raised when :attr:`plugin` is unknown."""
+
+    plugin: Plugin
+
+    def __init__(self, plugin: Plugin, *args: object) -> None:
+        super().__init__(*args)
+        self.plugin = plugin
+
+    def __str__(self) -> str:
+        return f"Unknown plugin '{self.plugin!r}' is not supported."
+
+
+class UnsupportedPluginVersionError(UnsupportedPluginError):
```

Comment:
I prefer to keep both exceptions since the cause of the exception is simply clear just from the name of the exception. Both will inherit from the `ValueError`.

Reply:
Looks good. I would change the text from "does not match" to "is outside of", but that's only another minor thing.